### PR TITLE
Updated installation instructions

### PR DESCRIPTION
**Pull request by Darylgolden**
## Changelog / Overview

- Added a warning about the incompatibility of different versions of Manim in the README
- Modified the admonition warning in the documentation
- Removed duplicated information from the README (`pip install manim` is already covered in the docs)

## Explanation for Changes

[Documentation Reference](https://manimce--####.org.readthedocs.build)

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,20 +34,15 @@ Manim is an animation engine for explanatory math videos. It's used to create pr
 -  [License](#license)
 
 ## Installation
-
+> **WARNING:** These instructions are for the community version _only_. Trying to use these instructions to install [3b1b/manim](https://github.com/3b1b/manim) or instructions there to install this version will cause problems. Read [this](https://docs.manim.community/en/stable/installation/versions.html) and decide which version you wish to install, then only follow the instructions for your desired version.
+
```
Comment (hydrobeam) [inline README.md:38]: - Given that the differences between manim versions are already listed under the docs, I don't think this particular section is necessary (maybe it should be made clearer there?). 

- Also, imo most people that are viewing the manimce repository will be trying to run code from it. If they're already here (most likely linked from 3b1b/manim) then this feels redundant.

```suggestion
```

Comment (Darylgolden) [reply README.md:38]: I think this is important enough to warrant reiterating, but I see your point. Let's see what others think.

Comment (jsonvillanueva) [reply README.md:38]: The trouble I see here is that the 3b1b/manim README.md isn't giving the same warning so without cooperation, we'd still likely face people using 3b1b/manim and running examples from the ManimCommunity documentation if somehow they ended up here.

I think it's enough to have this solely in our documentation and make it more clear there if we're still facing issues with people using incompatible code with the different versions of Manim. Somehow people are still using v0.1.1 documentation and that's more of an issue, but we already have a note/warning for viewing older versions of the docs.

Comment (behackl) [reply README.md:38]: I think it is a good idea to add the information in the README as well; there is little harm with replicating this information.

**Review by behackl — APPROVED** [review]
LGTM

**Comment by hydrobeam** [comment]
Based on some dicussion on this here; https://discord.com/channels/581738731934056449/867365098368270357/867556381258022932

we all agreed that this would be better suited as admonotion in the docs installation page.

> WARNING: These instructions are for the community version only. Trying to use these instructions to install [3b1b/manim](https://github.com/3b1b/manim) or instructions there to install this version will cause problems. Read [this](https://docs.manim.community/en/stable/installation/versions.html) and decide which version you wish to install, then only follow the instructions for your desired version.

I guess it's been stalled a bit

**Comment by behackl** [comment]
> Based on some dicussion on this here; https://discord.com/channels/581738731934056449/867365098368270357/867556381258022932
> 
> we all agreed that this would be better suited as admonotion in the docs installation page.
> 
> > WARNING: These instructions are for the community version only. Trying to use these instructions to install [3b1b/manim](https://github.com/3b1b/manim) or instructions there to install this version will cause problems. Read [this](https://docs.manim.community/en/stable/installation/versions.html) and decide which version you wish to install, then only follow the instructions for your desired version.
> 
> I guess it's been stalled a bit

We do have a corresponding information admonition right on top of the installation page. I still think this should be added to the README as well.

**Comment by jsonvillanueva** [comment]
I don't really mind it being in the README; although I do think it's not needed if the admonition is doing its job. The note admonition should be changed to a warning though -- I'll push a commit momentarily.

**Pull request merged**